book/src/concepts/register_types.md

```diff
@@ -1,38 +0,0 @@
-<!--
```

Comment:
Right, it might make sense to re-add it if we allow you to specify a default for qml_uri and friends, so that we have a place to explain the different options (qml_elemnt, qml_uri, qml_version, etc.).

But fine to delete for now.